support/controlplane-component/controlplane-component.go

```diff
@@ -40,6 +41,7 @@ type ControlPlaneContext struct {
 	ReleaseImageProvider     imageprovider.ReleaseImageProvider
 	UserReleaseImageProvider imageprovider.ReleaseImageProvider
 	ImageMetadataProvider    util.ImageMetadataProvider
+	DigestLister             registryclient.DigestListerFN
```

Comment:
can we document DigestLister and the other Public variables here

Reply:
done